tests/functional/test_builtin.py

```diff
@@ -117,3 +117,82 @@ def test_jsonarray2netcdf_execute(self):
         assert isinstance(nc_path, str) and len(nc_path)
         assert nc_path.startswith(wps_out)
         assert os.path.split(nc_real_path)[-1] == os.path.split(nc_path)[-1]
+
+    def test_file2string_array_describe(self):
+        resp = self.app.get("/processes/file2string_array", headers=self.json_headers)
+        assert resp.status_code == 200
+        assert resp.content_type in CONTENT_TYPE_APP_JSON
+        assert resp.json["process"]["id"] == "file2string_array"
```

Comment:
Out of scope for this PR, but maybe we could move all the "conformance" stuff to a single util function, so that if (when) the API is updated it's easier to update all the tests at once rather than every single test function.

Reply:
Yes. Big refactoring could be considered about that. 
But at the same time, it wasn't that big of a problem, and I don't expect the format to change much more since OGC seems to stabilize. 
I did most of the conversion in PR #261